numba/cuda/tests/cudadrv/data/cuda_include.cu

```diff
@@ -0,0 +1,5 @@
+// Not all CUDA includes are safe to include in device code compiled by NVRTC,
```

Comment:
I think these `.cu` files will need adding to `package_data` in `setup.py`.

Reply:
Added.